ffmpeg-sys-the-third/build.rs

```diff
@@ -769,17 +769,15 @@ fn check_features(include_paths: &[PathBuf]) {
         }
     }
 
-    let version_check_info = [("avcodec", 57, 62, 0, 101)];
-    for &(lib, begin_version_major, end_version_major, begin_version_minor, end_version_minor) in
-        &version_check_info
-    {
-        let libversion = *versions
-            .get(lib)
-            .expect("Unable to find the version for lib{lib}");
-
+    let begin_version_major = 57;
+    let end_version_major = 62;
+    let begin_version_minor = 0;
+    let end_version_minor = 134;
```

Comment:
We should probably just expand the `version_check_info` array instead.. Using these versions for all libraries means that
- some libraries get skipped (libavfilter, libswscale, libswresample) because their versions are outside of this range
- we get even more redundant `library_version_greater_than_x_y` features than we have at the moment

I'd appreciate if you could go through the trouble of doing this.

Reply:
Ok, added the `min_major` to `Library` struct and looped over its range bdc65868e76f241bb88e89c69464156c80bea51a

Reply:
I guess that works. There's probably no easy way to avoid this sort of "greater_than_51_0" through to "greater_than_51_135" spam when the actual version is 61.Y.